judo-runtime-core-dispatcher/src/main/java/hu/blackbelt/judo/runtime/core/dispatcher/behaviours/CreateInstanceCall.java

```diff
@@ -53,23 +64,55 @@ public boolean isSuitableForOperation(EOperation operation) {
+        CallInterceptorUtil<CreateInstanceCallPayload, Payload> callInterceptorUtil = new CallInterceptorUtil<>(
+                CreateInstanceCallPayload.class, Payload.class, asmModel, operation, interceptorProvider);
+
         final EReference owner = (EReference) asmUtils.getOwnerOfOperationWithDefaultBehaviour(operation)
                 .orElseThrow(() -> new IllegalArgumentException("Invalid model"));
 
-        final String inputParameterName = operation.getEParameters().stream().map(p -> p.getName()).findFirst()
+        final String inputParameterName = operation.getEParameters().stream().map(ENamedElement::getName).findFirst()
                 .orElseThrow(() -> new IllegalArgumentException("Input parameter name must be defined"));
 
-        final boolean bound = AsmUtils.isBound(operation);
-
-        if (AsmUtils.annotatedAsTrue(owner, "access") || !asmUtils.isMappedTransferObjectType(owner.getEContainingClass())) {
-            checkArgument(!bound, "Operation must be unbound");
-            return dao.create(owner.getEReferenceType(),
-                    asPayload((Map<String, Object>) exchange.get(inputParameterName)), null);
-        } else {
-            checkArgument(bound, "Operation must be bound");
-            return dao.createNavigationInstanceAt((ID) exchange.get(identifierProvider.getName()),
-                    owner,
-                    asPayload((Map<String, Object>) exchange.get(inputParameterName)), null);
+        CreateInstanceCallPayload inputParameter =
+                callInterceptorUtil.preCallInterceptors(
+                        CreateInstanceCallPayload.builder()
+                                .owner(owner)
+                                .instance(Payload.asPayload(exchange))
+                                .input(asPayload((Map<String, Object>) exchange.get(inputParameterName)))
+                                .build());
+
+        Payload ret = null;
+        if (callInterceptorUtil.isOriginalCalled()) {
+            final boolean bound = AsmUtils.isBound(operation);
+
+            if (AsmUtils.annotatedAsTrue(inputParameter.getOwner(), "access") ||
+                    !asmUtils.isMappedTransferObjectType(owner.getEContainingClass())) {
+                checkArgument(!bound, "Operation must be unbound");
+                ret = dao.create(
+                        inputParameter.getOwner().getEReferenceType(),
+                        inputParameter.getInput(), null);
+            } else {
+                checkArgument(bound, "Operation must be bound");
+                ret = dao.createNavigationInstanceAt(
+                        (ID) inputParameter.getInstance().get(identifierProvider.getName()),
+                        inputParameter.getOwner(),
+                        inputParameter.getInput(), null);
+            }
         }
+        return callInterceptorUtil.postCallInterceptors(inputParameter, ret);
```

Comment:
As per documented in `OperationCallInterceptor.java` if `(callInterceptorUtil.isOriginalCalled()` is `false` the post methods should also not be triggered.

Reply:
The documentation in interface have to be changed.